lib/mongo/auth/scram256/conversation.rb

```diff
@@ -0,0 +1,509 @@
+# Copyright (C) 2014 MongoDB Inc.
```

Comment:
This was copied from lib/mongo/auth/scram/conversation.rb and modified as necessary.